ui_framework/components/info_button/_info_button.scss

```diff
@@ -0,0 +1,13 @@
+.kuiInfoButton {
+  font-size: 16px;
+  background-color: transparent;
+  color: $infoButtonColor;
+  cursor: pointer;
+  border: none;
+  padding: 0;
+
+  &:hover,
+  &:active {
+    color: $infoButtonColor;
```

Comment:
Sorry I didn't mention this earlier, but what do you think of using our existing link colors?

```scss
.kuiInfoButton {
  font-size: 16px;
  background-color: transparent;
  color: $globalLinkColor;
  cursor: pointer;
  border: none;
  padding: 0;

  &:hover,
  &:active {
    color: $globalLinkColor-isHover;
  }
}
```